components/base/client_test.go

```diff
@@ -42,8 +42,11 @@ func setupWorkingBase(
 		return nil
 	}
 
-	workingBase.WidthFunc = func(ctx context.Context) (int, error) {
-		return width, nil
+	workingBase.PropertiesFunc = func(ctx context.Context, extra map[string]interface{}) (map[base.Feature]float64, error) {
+		return map[base.Feature]float64{
+			base.TurningRadiusM: 1.2,
+			base.WidthM:         0.6,
```

Comment:
The old version mentioned the variable `width`, but the new one doesn't seem to use that at all. If that's on purpose, can we remove the `width` argument from `setupWorkingBase`? and if that's not on purpose, please put it back.

Reply:
Not the `Width` function, the `width` argument (an integer) to the `setupWorkingBase` function.

Reply:
ah I misunderstood, let me see what that code is doing.

Reply:
changed it to still take a width, but return that width in Properties as  meters for this test.